components/org.wso2.carbon.identity.application.authenticator.hypr.rest/org.wso2.carbon.identity.application.authenticator.hypr.rest.dispatcher/pom.xml

```diff
@@ -159,17 +113,12 @@
         <dependency>
             <groupId>org.wso2.carbon.identity.outbound.auth.hypr</groupId>
             <artifactId>org.wso2.carbon.identity.application.authenticator.hypr.rest.v1</artifactId>
-            <version>1.0.0-SNAPSHOT</version>
+            <version>${project.version}</version>
```

Comment:
Version can remove from here and can move to parent pom

Reply:
Resolved via [96ebd76](https://github.com/wso2-extensions/identity-outbound-auth-hypr/pull/3/commits/96ebd76de74bc5703b401f46b9a35f1ae9812286)